game/src/AI/Steering/Behaviors/Individual/FollowHeatmapBehavior2D.gd

```diff
@@ -0,0 +1,36 @@
+extends SteeringBehavior2D
+class_name FollowHeatmapBehavior2D
+"""
+A 2D steering behavior that expects a heatmap to be somewhere in the scene.
+It uses that to figure out where it should be going - from large values in its cell
+towards smaller values, with 0 being the goal.
+"""
+
+
+var _heatmap #should be able to do `var _heatmap :Heatmap` here because the GDNS file
```

Comment:
Are you sure you can use gdnative types as gdscript types in Godot 3.1?

Reply:
I guess I was under the impression that class_name was for identifying the class so it could be used elsewhere, but it seems it's there just to identify which class inside of the binary it refers to.

Hopefully that gets improved in future NativeScript versions.